controller/workitem.go

```diff
@@ -111,18 +136,44 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	if !authorized {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not authorized to access the space"))
 	}
+
+	if ctx.Payload.Data.Relationships != nil && ctx.Payload.Data.Relationships.BaseType != nil &&
+		ctx.Payload.Data.Relationships.BaseType.Data.ID != wi.Type {
+		// Store new values of type and version
+		newTypeID := ctx.Payload.Data.Relationships.BaseType
```

Comment:
[SUGGESTION] `ctx.Payload.Data.Relationships.BaseType` is of type `*relationBaseType` and not uuid, so may be the variable name should be `newType` instead of `newTypeID`

Reply:
Done https://github.com/fabric8-services/fabric8-wit/pull/2202/commits/a4f1106aeb9b663421a6297d46f4a4d6ea719aee